cmd/osv-scanner/main_test.go

```diff
@@ -1071,6 +1107,287 @@ func TestRun_LocalDatabases(t *testing.T) {
 	}
 }
 
+func TestRun_Licenses(t *testing.T) {
+	t.Parallel()
+	tests := []cliTestCase{
```

Comment:
Can you add one test for markdowntable output?

Reply:
done